StackExchange/Stacks
# chore(opacity): bump disabled opacity to 0.55

[STACKS-590](https://stackoverflow.atlassian.net/browse/STACKS-590)

--- 

This PR does one thing: changes the value of the disabled custom properties used across Stacks from `0.5` to `0.55`. That is all!

**Acceptance Criteria**

- Bump the opacity for disabled button to 55%
- ~~Explore if we can test disabled button for a11y  color contrast in Stacks Classic~~ Moved to 

**Before**

<img width="802" alt="Screenshot 2024-11-28 at 8 23 27 AM" src="https://github.com/user-attachments/assets/ced73bb6-7c6d-4ce6-8471-4cd4d60c01be">

---

**After (very subtle)**

<img width="805" alt="Screenshot 2024-11-28 at 8 25 07 AM" src="https://github.com/user-attachments/assets/6dbaf3be-7d87-4d2a-8eef-c0af6dd652b9">

## How to test

- Go to any docs page that shows examples of disabled elements ([buttons](https://deploy-preview-1860--stacks.netlify.app/product/components/buttons/#disabled), [inputs](https://deploy-preview-1860--stacks.netlify.app/product/components/inputs/#base-style))
- Verify that the elements `opacity` value is now set to `0.55` (via the `--_o-disabled-static` or `--_o-disabled` custom property)
